README.md

```diff
@@ -659,6 +651,7 @@ This application is built with the following principles.
         5. UI updates with data from the server
 
 1. **Cross Platform 99.9999%**
+This section has moved [here](contributingGuides/philosophies/CROSS-PLATFORM.md).
     1. A feature isn't done until it works on all platforms.  Accordingly, don't even bother writing a platform-specific code block because you're just going to need to undo it.
     1. If the reason you can't write cross-platform code is because there is a bug in ReactNative that is preventing it from working, the correct action is to fix RN and submit a PR upstream -- not to hack around RN bugs with platform-specific code paths.
     1. If there is a feature that simply doesn't exist on all platforms and thus doesn't exist in RN, rather than doing if (platform=iOS) { }, instead write a "shim" library that is implemented with NOOPs on the other platforms.  For example, rather than injecting platform-specific multi-tab code (which can only work on browsers, because it's the only platform with multiple tabs), write a TabManager class that just is NOOP for non-browser platforms.  This encapsulates the platform-specific code into a platform library, rather than sprinkling through the business logic.
```

Comment:
Do we want to remove these points from here considering it is also in the new CROSS-PLATFORM.md file?